src/mobile/nodejs-assets/nodejs-project/main.js

```diff
@@ -43,13 +47,18 @@ const createSeedVault = async (seed, password) => {
  *
  * @param {arrayBuffer} Db - Encrypted binary KDBX database
  * @param {string} Password - Password for decryption
- * @returns {array} Decrypted seed byte array
+ * @returns {string} Serialised decrypted seed byte array and title
  */
 const getSeedFromVault = async (buffer, password) => {
     const credentials = new kdbxweb.Credentials(kdbxweb.ProtectedValue.fromString(password));
     const db = await kdbxweb.Kdbx.load(buffer, credentials);
-    const seed = db.getDefaultGroup().entries[0].fields.Seed.getText();
-    return seed;
+
+    const entry = db.getDefaultGroup().entries[0];
+
+    const seed = entry.fields.Seed.getText();
+    const title = entry.fields.Title;
+
+    return JSON.stringify({ seed, title });
```

Comment:
Does stringifying the seed defeat the purpose of having it in a buffer? Maybe we should just use the desktop implementation: https://github.com/iotaledger/trinity-wallet/blob/develop/src/desktop/native/kdbx.js

Reply:
The string is already passed to rn-nodejs events as a string.